docs/index.md

```diff
@@ -116,6 +116,9 @@ up a Python {{ python_version }} environment with `conda`:
 
 ### From conda
 
+Installation from conda-forge using conda or mamba is recommended for arm64 macOS machines
```

Comment:
```suggestion
On newer macOS machines with an arm64 architecture, installation using conda is required, because it is not possible to get all the pre-compiled `napari` dependencies from `pip`:
```
I think "required" instead of "recommended" is better here. Ideally the instructions should provide **one** way for the user to install `napari`, instead of them having to guess between different possible ways themselves.